vana/state.py

```diff
@@ -84,6 +84,27 @@ def __init__(
         if sync:
             self.sync(block=None, lite=lite)
 
+        self.cache_file = f"state_cache_{network}_{dlp_uid}.json"
+
+    def save_cache(self):
```

Comment:
What's the difference between the cache and the state file? Seems to be saving the same data?